cellcounter/cc_kapi/models.py

```diff
@@ -40,29 +62,35 @@ def set_keymaps(self, new_mapping_list):
         """new_mapping_list is a list of KeyMap objects"""
         self._sync_keymaps(new_mapping_list)
 
-    def set_primary(self):
-        self._sync_primary_flags()
-
     def delete(self):
-        """Custom delete method, covering setting primary flags"""
+        """Custom delete method, covering setting default flags"""
+        # FIXME: get the DefaultKeyboards to delete the default
```

Comment:
Fixed in https://github.com/cellcounter/cellcounter/pull/446/commits/96df3a4f598321e1c24de49343b66e13f2c76d68